### ISSUE TITLE
Get rid of redundant accessor functions

### ISSUE THREAD

**Issue by lehins** [IntersectMBO/cardano-ledger#3130]
Every era defines a whole bunch of accessor functions eg, `inputs'`, `outputs'`, etc. which are redundant and should be removed, while all use sites should be switched to corresponding lenses. In case of ShelleyMA it even defines a pattern synonym. That one should be removed as well

**Comment by aniketd** [comment]
> In case of ShelleyMA it even defines a pattern synonym. That one should be removed as well

This part has been addressed in #3126

**Comment by lehins** [comment]
They should first be deprecated and then we can get rid of them at some point later.
Some parts have been addressed in https://github.com/IntersectMBO/cardano-ledger/pull/4973
but there are still synonyms like that lingering around.

**Comment by neilmayhew** [comment]
The only remaining undeprecated accessor functions I was able to find were in Alonzo, and I couldn't find any redundant patterns. If there are any that I've missed, please let me know.

**Comment by lehins** [comment]
> The only remaining undeprecated accessor functions I was able to find were in Alonzo, and I couldn't find any redundant patterns.

Yep, looking for those fields to deprecate was part of this ticket. I also think that it was only in Alonzo that we had those left over. Thank you for double checking.
